frontend/src/components/Charts/CandlestickChart.ts

```diff
@@ -7,11 +7,7 @@ import * as dayjs from 'dayjs'
 import * as duration from 'dayjs/plugin/duration'
 import {AssetSymbol} from '../../../../common/models'
 
-export type CandlestickSeries = {
-    x: Date,
-    y: [open: number, high: number, low: number, close: number]
-}
-
+export type CandleData = { date: dayjs.Dayjs | undefined; open: any; high: any; low: any; close: any; volume: any }
```

Comment:
Is the type really any here? Why not number or number|string?

Reply:
Nah, they should be all `number`